app/views/directives/edit-environment-from.html

```diff
@@ -61,6 +61,9 @@
               </ui-select-choices>
             </ui-select>
           </div>
+          <div class="has-warning" ng-show="$ctrl.hasInvalidEnvVar(entry.selectedEnvFrom.data)">
```

Comment:
Any reason not to use `ng-if`  instead of `ng-show`?

Reply:
Was looking on https://github.com/openshift/origin-web-console/blob/master/app/views/directives/osc-routing.html#L27 as an example no which attr are we using, but Now I've noticed that we are mostly using `ng-show`. Sorry for that